### ISSUE TITLE
Create a CNN to Classify Dog Breeds (from Scratch)

### ISSUE THREAD

**Issue by ierolsen** [ierolsen/Udacity-Dog-Breed-Classifier#3]
## Specify DataLoaders for the Dog Dataset
Write three separate data loaders for the training, validation, and test datasets of dog images. These images should be pre-processed to be of the correct size.

## **Question 3:** Describe your chosen procedure for preprocessing the data.
Answer describes how the images were pre-processed and/or augmented.

## **Model Architecture**
The submission specifies a CNN architecture.

## **Question 4:** Outline the steps you took to get to your final CNN architecture and your reasoning at each step.
Answer describes the reasoning behind the selection of layer types.

## **Train the Model**
Choose appropriate loss and optimization functions for this classification task. Train the model for a number of epochs and save the "best" result.

## **Test the Model**
The trained model attains at least 10% accuracy on the test set.
